apps/take_picture/app.c

```diff
@@ -0,0 +1,197 @@
+/**
+ * This example takes a picture every 5s.
+   Example src: https://github.com/espressif/esp32-camera.git 
+ */
+
+#define BOARD_WROVER_KIT
+#define BOARD_ESP32CAM_AITHINKER
```

Comment:
Is it possible to add an option in `menuconfig` where one board is selected?

Reply:
According to espressif the only difference is in the sdkconfig.defaults file.
In this line : CONFIG_ESP32_SPIRAM_SUPPORT=y
the line can be added menually like I did or it can be added in the menuconfig (Enable PSRAM). 
Src: https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-guides/build-system.html#selecting-idf-target
and https://github.com/espressif/esp32-camera

Reply:
Sorry I should have been more precise.
What I mean is that there are two defines here `#define BOARD_WROVER_KIT` and `#define BOARD_ESP32CAM_AITHINKER` they define which camera pinout to use (see[ Lines 35-79](https://github.com/micro-ROS/freertos_apps/blob/54e2aae47980d935a9e7e99c317dea4bc2bd8876/apps/take_picture/app.c#L35-L79) ) and should be mutually exclusive.
Therefore I suggest to add an option to menuconfig where one of the two can be selected, which would also make it easier to add a new camera pinout for example the ESP-EYE development board.

Reply:
check my last commit I added the option to choose camera pinout.